modules/KalturaSupport/components/sourceSelector.js

```diff
@@ -142,9 +142,8 @@
                 this.getPlayer().mediaElement.removeSourceFlavor(sources);
             }
 
-			if( sources.length == 1 ){
-				// no need to do building menu logic. 
-				this.addSourceToMenu( sources[0], _this.getSourceTitle(sources[0]) );
+			if( sources.length == 1 && this.getPlayer().streamerType != "http" ){
```

Comment:
Please explain the condition

Reply:
Sure, if the source length is 1 and if it's adaptive. 
If we are on progressive we don't add the "Auto" tag to he menu.